openassessment/xblock/static/js/src/lms/oa_response.js

```diff
@@ -941,4 +952,66 @@ OpenAssessment.ResponseView.prototype = {
             return url;
         });
     },
+
+    /**
+     * Generates HTML to display team information
+     *
+     * @param {string} teamName - The name of the team
+     * @param {array} usernames - The list of users on the team
+     */
+    generateTeamStringHTML: function(teamName, usernames) {
+        var userString = usernames.join('</strong>, <strong class="emphasis">');
+        var teamMsg;
+        if (usernames.length >= 1) {
+            userString = '<strong class="emphasis">' + userString + '</strong>';
+            teamMsg = 'Team Members: ' + userString;
+        } else {
+            teamMsg = 'You have no teammates';
```

Comment:
I think this is an impossible state.  If you're the only one on the team, `usernames` will only contain you.  If you're not in the team, `getTeamInfo()` will return null and you wouldn't expect to see any team info rendered.

Reply:
That ... is definitely true lol